README.md

```diff
@@ -1,70 +1,174 @@
+  - [**Contributing 🤝**](#contributing-)
+  - [**Show your support 🌟**](#show-your-support-)
+  - [**Acknowledgments 🙏**](#acknowledgments-)
+  - [License 📝](#license-)
+
+---
+<!-- BUILT WITH -->
+## **Built With 🛠**<a name="built-with"></a><br>
+
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML5</a> markup to build raw structure of this web page</li><br>
+    <li><a href="https://react.dev/">React</a></li><br>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS3</a> custom properties, <a href="https://css-tricks.com/snippets/css/a-guide-to-flexbox/">FlexBox</a>, <a href="https://www.w3schools.com/css/css_grid.asp">Grid</a> to make the website visually attractive</li><br>
+    <li><a href="https://reactjs.org/">Javascript</a></li><br>
+    <li><a href="https://mozilla.github.io/addons-linter/">Linters</a> for coding convention and coding formating</li>
+    <li><a href="https://webpack.js.org">Webpack</a></li>
+  </ul>
+
+<br>
+
+---
+<!-- KEY FEATURES -->
+### **Key Features 🏷️** <a name="key-features"></a>
+
+This capstone app displays metrics about the air pollution of a given city.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<br>
+
+---
+<!-- PREREQUISITIES -->
+## **Prerequisites 🧱**<a name="prerequisites"></a>
+
+In order to run this project you need:
+
+- A Github account
+- Node JS
+- Webpack
+- A web browser to view output e.g [Microsoft Edge](https://www.microsoft.com/en-us/edge).
+- An IDE e.g [Visual studio code](https://code.visualstudio.com/).
+- [A terminal](https://code.visualstudio.com/docs/terminal/basics).
+
+<br>
+
+---
+<!-- SETUP -->
+## **Setup ⚙️**<a name="setup"></a>
+
+Clone this repository:<br>
+```sh
+  git clone https://github.com/mariegrace31/react_capstone.git
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+```
+Navigate to the folder
+```
+cd react_capstone
+```
+Checkout the branch
+```
+git checkout main
+```
+<br>
 
-## Available Scripts
+---
+<!-- INSTALL -->
+## ** 🏗️**<a name="install"></a>
 
-In the project directory, you can run:
+Install all dependencies:
 
-### `npm start`
+```sh
+    npm install
+```
+Run the following code to fixed possible JavaScript linter errors:
+```
+npx eslint . --fix
+```
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+---
+<!-- USAGE -->
+## **Usage 📂**<a name="usage"></a>
+Open page in browser
+```sh
+$ npm start
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+---
+<!-- Future features -->
+## **Future features**<a name="Future-fetures"></a>
+-Feature 1: deploy my app
 
-### `npm test`
+VIDEO RECORDING:https://www.loom.com/share/2221bc4c38d14d5cacbb2c09afe98e56?sid=c1da20ce-4cf0-4064-b19b-6a2e4815635a
+LIVE DEMO: https://myreactcapstone.netlify.app/
```

Comment:
- [ ] Dear @mariegrace31, for the video presentation, though your presentation is great we still need to navigate to the code/vscode and show a piece of the code that you found interesting. Please record a video containing all the details you have now and show some code bases you find interesting.
- [Optional] Also, Please try to update your links with some placeholder text, this makes your readme file more professional.